src/msg/async/Event.cc

```diff
@@ -255,6 +255,7 @@ void EventCenter::delete_time_event(uint64_t id)
   }
 
   time_events.erase(it->second);
+  event_map.erase(it);
```

Comment:
could you make this a dedicated commit or at least note this in the commit message?